### DISCUSSION TITLE
Extending system values

### DISCUSSION THREAD

**Discussion by zslabs** [chakra-ui/panda#1726] [Q&A] [answered]
As part of https://github.com/chakra-ui/panda/discussions/1713 when using `strickTokens` - it mentions catching typos in properties that are defined in your config.

```ts
// config.strictTokens = true
css({ display: 'flex' }) // OK, didn't throw
css({ display: 'block' }) // OK, didn't throw
css({ display: 'abc' }) // ✅ will throw since 'abc' is not a valid value for 'display'
```

What I'm struggling with is how we further extend properties like `scale` or `willChange` in the config - as they don't appear to be listed as tokens. If this is possible; is there a spot in the docs I missed (or that should be added)? An example would be adding support for `willChange: 'transform'`.

```
SystemProperties.willChange?: ConditionalValue<Globals | "auto" | "contents" | "scroll-position"> | undefined
```

**Comment by astahmer** [comment, marked as answer]
![Screenshot 2023-11-30 at 10 21 15](https://github.com/chakra-ui/panda/assets/47224540/0bd94820-9549-4270-b0e2-2b897a67d577)

you can use utilities for that
https://play.panda-css.com/FKwNghGjRK

**Reply by zslabs** [reply]
Ah! That was what I was missing. Thank you very much!
